docs/docs/sdk-examples/introduction.mdx

```diff
@@ -0,0 +1,180 @@
+---
+title: Introduction
+sidebar_label: Introduction
+---
+
+You can integrate ZITADEL quickly into your application and be up and running within minutes.
+To achieve your goals as fast as possible, we provide you with SDKs, Example Repositories and Guides.
+
+The SDKs and Integration depend on the framework and language you are using.
+
+import { Tile } from "../../src/components/tile";
+
+### Resources
+
+<div className="tile-wrapper">
+<Tile title="Angular"
+    imageSource="/docs/img/tech/angular.svg"
+    link="/docs/sdk-examples/angular"></Tile>
+<Tile title="Flutter"
+    imageSource="/docs/img/tech/flutter.svg"
+    link="/docs/sdk-examples/flutter"></Tile>
+
+<Tile title="Go"
+    imageSource="/docs/img/tech/golang.svg"
+    link="/docs/sdk-examples/go"></Tile>
+<Tile
+    title="Java"
+    imageSource="/docs/img/tech/java.svg"
+    link="/docs/sdk-examples/java"></Tile>
+<Tile
+    title="NestJS"
+    imageSource="/docs/img/tech/nestjs.svg"
+    link="/docs/sdk-examples/nestjs"></Tile>
+<Tile
+    title="Next.js"
+    imageSource="/docs/img/tech/nextjs.svg"
+    link="/docs/sdk-examples/nextjs"></Tile>
+<Tile
+    title="Python"
+    imageSource="/docs/img/tech/python.svg"
+    link="/docs/sdk-examples/python"></Tile>
```

Comment:
Shouldn't we call this section "Flask" in anticipation of the "Django" python guide and example?

They also have a logo: https://flask.palletsprojects.com/en/3.0.x/_images/flask-horizontal.png

Reply:
you are right I changed it and renamed to python flask.